ee_tests/Dockerfile.builder

```diff
@@ -51,17 +54,14 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
 #  && yum install -y firefox \
 #  && npm install -g karma-firefox-launcher
 
-RUN npm install -g  yarn jasmine-node protractor
-
-WORKDIR /opt/fabric8-test/
-
-COPY . .
-
-VOLUME /dist
-
 # Provide oc client to tests Clean up the test user account's resources in OpenShift Online
 RUN wget https://github.com/openshift/origin/releases/download/v1.5.0/openshift-origin-client-tools-v1.5.0-031cbe4-linux-64bit.tar.gz &&\
     tar -xzvf openshift-origin-client-tools-v1.5.0-031cbe4-linux-64bit.tar.gz &&\
     mv openshift-origin-client-tools-v1.5.0-031cbe4-linux-64bit/oc oc
 
-RUN yarn
+# install all dependencies
+COPY package.json package-lock.json /opt/fabric8-test/
+RUN npm install && npm run webdriver:update
```

Comment:
Why is `npm run webdriver:update` required? Why doesn't it run using postinstall script?